### PR TITLE
fix loosing diffs in diff poster script

### PR DESCRIPTION
**Pull request by nikandfor**
The problem was that shell expanded this query to some its subset depending on the files if found.
`solutions/**/diff/**.diff`

### CODE REVIEW

File: scripts/post_diffs_in_pr.sh [hunk 1]
```diff
@@ -139,6 +146,12 @@ function delete_comment {
 function make_comment {
 	comment_id=`find_comment`
 
+	echo git status
+	git log --pretty=oneline --graph -20
+
+	has_diffs_current && echo "diffs found" || echo "no diffs found"
+	git diff --name-status --diff-filter=ACDM "$GITHUB_BASE_REF_SHA" "$GITHUB_REF_SHA" 'solutions/**/diff/**.diff' || true
```
Comment (rohitpaulk) [inline scripts/post_diffs_in_pr.sh:153]: @nikandfor what does this line do? Is it just for debugging? And if so, isn't the same output already printed from `has_diffs_current`?

**Review by rohitpaulk — APPROVED** [review]

File: scripts/post_diffs_in_pr.sh (hunk 1 above)
Comment (rohitpaulk) [reply scripts/post_diffs_in_pr.sh:153]: (don't need to block merging, I was just curious - PR approved)

Comment (nikandfor) [reply scripts/post_diffs_in_pr.sh:153]: has_diffs_current is a function which returns bool and print nothing. And here is debug printing. I could probably combine them, but I wanted to separate two tasks to have more flexibility.

Comment (rohitpaulk) [reply scripts/post_diffs_in_pr.sh:153]: Got it - makes sense!

**Pull request merged**